### ISSUE TITLE
RBAC: Groups don't have access to kubernetes API resources

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#9150]
**Rancher versions:**
rancher/server: v1.6.3-rc2
kubernetes: rancher/k8s:v1.6.6-rancher1-4

**Steps to Reproduce:**

- Install kubernetes with RBAC enabled
- Enable Github authentication
- Add the following clusterRole and clusterRoleBinding
```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1alpha1
metadata:
  name: default-reader
rules:
  - apiGroups: [""]
    resources:
      - componentstatuses
      - events
      - endpoints
      - namespaces
      - nodes
      - pods
      - persistentvolumes
      - resourcequotas
      - services
    verbs: ["get", "watch", "list"]
  - nonResourceURLs: ["*"]
    verbs: ["get", "watch", "list"]

---

kind: ClusterRoleBinding
apiVersion: rbac.authorization.k8s.io/v1alpha1
metadata:
  name: default-reader-role-binding
subjects:
  - kind: Group
    name: "github_org:rancher"
roleRef:
  kind: ClusterRole
  name: default-reader
  apiGroup: rbac.authorization.k8s.io
```
The previous role will give certain permissions to rancher group.
- Add the group to the environment users as a member
- Login with a user that belong to the group.
- try to query the api
```
$ kubectl get nodes
```

**Results:**
The user failed to query the api resources.

**Comment by deniseschannon** [comment]
Available in `k8s-1.7-dev` using `rancher/kubernetes-auth:v0.0.5`

**Comment by galal-hussein** [comment]
Rancher server: v1.6-development

By following the steps mentioned above, i was able to access the mentioned resources, using a user that belongs to the group in rbac access.
